app/(main)/login/page.tsx

```diff
@@ -66,12 +79,13 @@ const Login = (): JSX.Element => {
   });
 
   /**
-   * A function that handles form submission.
-   * @param {LoginUserSchemaType} data - The data submitted in the form.
-   * @returns {Promise<void>} A promise that resolves when the `login` function completes.
+   * Handles the form submission.
+   * @param {LoginUserSchemaType} data - The data from the form.
    */
   const onSubmit: SubmitHandler<LoginUserSchemaType> = async (data) => {
```

Comment:
We do need a unit test for this. Work with @vmaineng. She has a solution on testing loading elements.

Reply:
@shashilo I believe this comment is in regards to the global spinner I implemented recently with positive and negative expectations. If so, @ryandotfurrer , please see examples: https://github.com/LetsGetTechnical/gridiron-survivor/pull/454/files. I hope this helps :)